staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -152,6 +152,9 @@ type GenericAPIServer struct {
 
 	// delegationTarget is the next delegate in the chain or nil
 	delegationTarget DelegationTarget
+
+	// HandlerChainWaitGroup allows you to wait for all chain handlers exit after the server shutdown.
+	HandlerChainWaitGroup *sync.WaitGroup
```

Comment:
Shouldn't this be a non-pointer?

Reply:
No, because we have value copy.

Reply:
Where is GenericAPIServer copied such that we cannot keep this a non-pointer?

Reply:
It is copied in 
```
staging\src\k8s.io\apiserver\pkg\server\config.go 
func (c completedConfig) New(name string, delegationTarget DelegationTarget)
 (*GenericAPIServer, error) {
```
If use non-pointer, here should direct create sync.WaitGroup when GenericAPIServer init.
I can do this.

Reply:
It is copied in `staging\src\k8s.io\apiserver\pkg\server\config.go`,